test/dummy/config/initializers/other.rb

```diff
@@ -1,3 +1,5 @@
+Rails.application.config.assets.version = '1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.